### ISSUE TITLE
Can not find `.background()` When BackgroundView contains Color

### ISSUE THREAD

**Issue by amaocha-first** [nalexn/ViewInspector#169]
Environment
- ViewInspector 0.9.1
- Xcode 13.2.1
- iOS Deployment Target 15.0

The following test failed with error.
``` swift
func test_background_color() throws {
    let sut = Text("").background(Color.red)
    XCTAssertEqual(try sut.inspect().text().background().color().value(), Color.red)
}
```

error message: ```XCTAssertEqual failed: throw error "Text does not have `background` modifier."```

ViewInspector expect `sut` to contain a `_BackgroundModifier`. However, in fact, `sut` contains a `_BackgroundStyleModifier`.
In my case, when iOS Deployment Target is 14.0, the above test ends with success and `sut` contains a `BackgroundModifier`.

**Comment by nalexn** [comment]
Hey, thanks for reporting! Yep, this is clearly a change in iOS 15 that wasn't covered with tests in the lib. Should be an easy fix.

**Comment by nalexn** [comment]
Hey, I just got to look into your report, and I can see the test you've provided passes for both iOS 14 and 15 in Xcode 13.4. Could you re-test?

**Comment by nalexn** [comment]
This is fixed for the coming new version

**Comment by nalexn** [comment]
Released with v0.9.2
